2021-04-17 08:58:23,352: INFO
2021-04-17 08:58:29,768: INFO
2021-04-17 08:59:22,277: INFO
2021-04-17 08:59:30,275: ERROR
Traceback (most recent call last):
  File "C:/Users/karth/OneDrive/Daily/PycharmProjects/Logging/logmechanism.py", line 8, in <module>
    print('The result is :',x/y)
ZeroDivisionError: division by zero
2021-04-17 08:59:30,275: INFO
2021-04-17 09:01:35,376: INFO
2021-04-17 09:01:47,970: ERROR
Traceback (most recent call last):
  File "C:/Users/karth/OneDrive/Daily/PycharmProjects/Logging/logmechanism.py", line 7, in <module>
    y=int(input('Enter second number:'))
ValueError: invalid literal for int() with base 10: 'two'
2021-04-17 09:01:47,971: INFO
2021-04-17 15:08:14,415: INFO
2021-04-17 15:08:19,128: ERROR
Traceback (most recent call last):
  File "C:/Users/karth/OneDrive/Daily/PycharmProjects/Logging/logmechanism.py", line 8, in <module>
    print('The result is :',x/y)
ZeroDivisionError: division by zero
2021-04-17 15:08:19,129: INFO
